packages/react-charting/docs/TestPlans/HeatMapChart/ComponentTests.md

```diff
@@ -0,0 +1,30 @@
+# Heatmap Chart - Component test plan
+
+## Snapshot tests
+
+| Test case                                                                    | Validation                            | Tool used |
+| ---------------------------------------------------------------------------- | ------------------------------------- | --------- |
+| With only data, domainValuesForColorScale and rangeValuesForColorScale props | Should render heatmap chart correctly | Enzyme    |
+| When data is not present for some group                                      | Should render heatmap chart correctly | Enzyme    |
+| When hideLegend prop is set to true                                          | Should render heatmap chart correctly | Enzyme    |
+| When yAxisTickFormat prop is set                                             | Should render heatmap chart correctly | Enzyme    |
+| When the layout direction is RTL                                             | Should render heatmap chart correctly | Enzyme    |
+| When the theme is dark                                                       | Should render heatmap chart correctly | Enzyme    |
+| When custom formatter functions are set for x and y axis strings             | Should render heatmap chart correctly | RTL       |
```

Comment:
This would be date or numeric, not strings right?

Reply:
HeatMapChart uses format strings for number and date axis labels but formatter functions for string axis labels.

Reply:
Can you mention this issue and issue id in the test description. This is a bug and not an expected behavior.

Reply:
done